drivers/net/ethernet/intel/idpf/idpf_eth_idc.h

```diff
@@ -0,0 +1,199 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/* Copyright (C) 2024 Intel Corporation */
+
+#ifndef _IDPF_ETH_IDC_H_
+#define _IDPF_ETH_IDC_H_
+
+#include <linux/auxiliary_bus.h>
+#include "idpf_eth_common.h"
```

Comment:
This header should be independent as much as possible. what is the common header dependency??

Reply:
Done ! Moved the include file to idpf.h file.